buildSrc/src/main/kotlin/ai.smoke-test.gradle.kts

```diff
@@ -87,6 +87,21 @@ configurations.all {
 }
 
 tasks {
+  // FIXME (trask) copy-pasted from ai.java-conventions.gradle
```

Comment:
nit: Not sure what is the intent of this comment. Should this be removed or modified?